pkg/helm/helm_fake.go

```diff
@@ -1,125 +1,126 @@
 package helm
 
```

Comment:
If it's not needed then we can delete it in another PR so we dont have to trigger CI again

Reply:
Yeah I wasn't sure. It wasn't being used anywhere but I was a bit hesitant to delete it completely. I'll open a PR for it's removal.

Reply:
OK cool - @ccojocar you happy for this to go or was there a use planned for it?

Reply:
We need this :D. Why are you deleting? The idea is to use it in unit tests to simulate helm without executing any helm command.

Reply:
ok so it's more that there's no helm tests that use it yet?

Reply:
That's right. We plan to introduce it for teraform commands.

Reply:
Did you generate it using a mocking package or rolled it yourself? If we want to go full blown mocks it might be worth picking package for mocking and using it everywhere. Having something that plugs in to `go generate`would also be useful as it's then really easy to regenerate the mock when the interface changes. Then we don't have to write/maintain mocks ;-) Personally, unless we are looking to go full TDD I think that in many cases the approach I've used in my refactor of the existing helm tests might be a less obtrusive solution. Simply patch the methods you need to encapsulate the test.

Reply:
I am fine to remove it, if you think that is easy to generate a mock for this interface.